README.md

```diff
@@ -1,25 +1,49 @@
-# kubevirt-ansible
+# KubeVirt Ansible
 
 This repository provides collection of playbooks to
-* install KubeVirt on existing OpenShift cluster
-* deploy K8S on given machines and install KubeVirt
-* deploy OpenShift on given machines and install KubeVirt
+- [x] [install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-openshift-cluster)
+- [ ] deploy Kubernetes cluster on given machines and install KubeVirt
+- [x] [deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
+- [ ] deploy Kubernetes cluster with KubeVirt with Lago
+- [x] [deploy Kubernetes cluster with KubeVirt with Lago](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
 
-Tested on CentOS Linux release 7.3.1611 (Core), OpenShift 3.7 and Ansible 2.3.1
+**Tested on CentOS Linux release 7.4 (Core), OpenShift 3.7 and Ansible 2.4.2**
 
+## List of playbooks
+
+| Playbook name | Description |
+| ------------- | ----------- |
+| install-kubevirt-on-openshift.yml | This playbook installs KubeVirt on existing OpenShift cluster. |
+| deploy-kubernetes.yml | This playbook deploys Kubernetes cluster on given machines. |
+| deploy-openshift.yml | This playbook deploys OpenShift cluster on given machines. |
+| control.yml | This is top level playbook which deploys Kubernetes or OpenShift cluster using Lago provider and installs KubeVirt on top of it. |
+
+## Requirements
+
+Install depending roles, and export `ANSIBLE_ROLES_PATH`
+
+```bash
+$ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
+$ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
+```
+
+For OpenShift deployment clone [**OpenShift Ansible project**][openshift-ansible-project]
+
+```bash
+$ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
+```
 
 ## Install KubeVirt on existing OpenShift cluster
```

Comment:
Maybe we can structure the doucment a bit different.

Chapter: (Optional) Setup a cluster (openshift or kubernetes)

Next ChapteR: Deploy KubeVirt on a cluster (openshift or kubernetes)

Within the chapters we can then handle kubernetes or openshift.

because then the doc is somewhat in one flow: either use an existing or create a new cluster. then deploy kubevirt on the cluster.